Code/Framework/AzCore/Tests/AZStd/Lists.cpp

```diff
@@ -386,27 +384,27 @@ namespace UnitTest
         AZ_TEST_ASSERT(((AZStd::size_t)&int_list.front() % 4) == 0); // default int alignment
 
         // make sure every allocation is aligned.
-        list<MyClass> aligned_list(5, MyClass(99));
-        AZ_TEST_ASSERT(((AZStd::size_t)&aligned_list.front() & (alignment_of<MyClass>::value - 1)) == 0);
+        AZStd::list<UnitTestInternal::MyClass> aligned_list(5, UnitTestInternal::MyClass(99));
+        AZ_TEST_ASSERT(((AZStd::size_t)&aligned_list.front() & (AZStd::alignment_of<UnitTestInternal::MyClass>::value - 1)) == 0);
     }
 
     TEST_F(ListContainers, StaticBufferAllocator)
     {
         /////////////////////////////////////////////////////////////////////////
         // Test swap, splice, merge, etc. of containers with different allocators.
         // list operations with container with the same allocator.
-        typedef static_buffer_allocator<16*1024, 1> static_buffer_16KB_type;
+        using static_buffer_16KB_type = AZStd::static_buffer_allocator<16 * 1024, 1>;
         static_buffer_16KB_type myMemoryManager1;
         static_buffer_16KB_type myMemoryManager2;
-        typedef allocator_ref<static_buffer_16KB_type> static_allocator_ref_type;
+        using static_allocator_ref_type = AZStd::allocator_ref<static_buffer_16KB_type>;
         static_allocator_ref_type allocator1(myMemoryManager1, "Mystack allocator 1");
         static_allocator_ref_type allocator2(myMemoryManager2, "Mystack allocator 2");
 
-        typedef list<MyClass, static_allocator_ref_type> stack_myclass_list_type;
+        typedef AZStd::list<UnitTestInternal::MyClass, static_allocator_ref_type> stack_myclass_list_type;
```

Comment:
suggestion: Is it worth updating this to the more modern `using` syntax while we're here?

Reply:
I have missed this change.
I am updating these files to use the `using` syntax

Reply:
Nice one thanks!